edward/models/distributions.py

```diff
@@ -6,23 +6,37 @@
 from edward.util import cumprod, get_dims, get_session, to_simplex
 
 class Distribution:
-    """
-    Base class for distributions
+    """Base class for Edward model distributions.
```

Comment:
done (but this is something we may want to discuss at some point. we should distinguish between `edward.models.distributions` and `edward.stats.distributions`). it is confusing. perhaps we could call the latter `edward.prob` or something.